draft-brw-sconepro-rate-policy-discovery.md

```diff
@@ -883,6 +894,30 @@ This example assumes that the Network Access Server (NAS) embeds both Remote Aut
 ~~~~~
 {: #radius-ex title="An Example of RADIUS NRLP Exchanges"}
 
+# Alternative/Complementary Mechanisms
+
+In the event of bottlenecks in a network, there are other mechanisms that provide information or help to reserve resources. These can be used within the bottleneck network or, in some cases, across network boundaries. The following sections give examples of such mechanisms and provide background information.
+
+## L4S {#L4S}
+
+Low Latency, Low Loss, and Scalable Throughput (L4S) is an architecture defined in {{RFC9330}} to avoid queuing at bottlenecks by capacity-seeking congestion controllers of senders. L4S support addresses the investigated use case of this document, which considers rate limiting, which typically involves queuing discipline at the rate limiting bottleneck. If all involved elements (UE, network, and service) support L4S, the use of Explicit Congestion Notification (ECN) provides the measure used to inform the network protocol and/or service endpoints in use of impending congestion. Congestion detection and reaction may require some few RTTs, though to adjust to the network forwarding conditions.
+
+As of 3GPP Rel. 18 (5G Advanced, {{TS-23.501}}), L4S is also defined for the 5G system (5GS) and can be used by UE and its services, but also for external parties of the 5GS by exposure of congestion information.
+
+## Network Slicing {#ns}
+
+One measure for guaranteeing resources in networks is network slicing. This is achieved by configuring certain resources , including adequate QoS setup for communication streams, which are taken into account in packet schedulers along the transport path, e.g., the RAN air interface.
+
+Network slicing is considered by the 3GPP for 5G {{TS-23.501}} (an equivalent can be achieved in 4G by configuring QFI values), the IETF {{RFC9543}} for transport networks, and BBF {{TR-470}} for wireline access. A realization model in transport networks is detailed in {{?I-D.ietf-teas-5g-ns-ip-mpls}}.
+
+L4S {{L4S}} can be used for the realization of a network slice. Network slices properties (e.g., throughput) can be retrieved from an operator network by third parties via a network API {{network_api}} (e.g., 3GPP NEF).
```

Comment:
```suggestion
L4S {{L4S}} can be used for the realization of a network slice. Network slices properties (e.g., throughput) can be retrieved from an operator network or configured by third parties via a network API {{network_api}} (e.g., 3GPP NEF).
```
Shouldn't we emphasize also the configuration of network slices, e.g. by VoD providers, to achieve smooth playback?

Reply:
This can be mentioned as an example.